Libraries/LibWeb/HTML/HTMLDivElement.h

```diff
@@ -21,6 +21,8 @@ class HTMLDivElement : public HTMLElement {
     // https://www.w3.org/TR/html-aria/#el-div
     virtual Optional<ARIA::Role> default_role() const override { return ARIA::Role::generic; }
 
+    virtual bool is_html_div_element() const override { return true; }
```

Comment:
You state in your PR description that you see an improvement in e.g. scrolling. Can you share which page(s) you used to test this so we can try to reproduce the effect? Are you able to perform actual measurements (profiles, benchmarks, or other)?

Reply:
I tried with WSL (Ubuntu) on some GitHub pages like LB issues page, the GitHub homepage and Wikipedia. I'm unable to perform actual measurements unfortunately